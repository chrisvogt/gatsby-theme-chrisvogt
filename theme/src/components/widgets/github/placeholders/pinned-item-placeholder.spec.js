import PinnedItemPlaceholder from './pinned-item-placeholder'

describe('Widgets/GitHub/PinnedItemPlaceholder', () => {
  it('matches the snapshot', () => {
    expect(PinnedItemPlaceholder).toMatchSnapshot()
  })
})
