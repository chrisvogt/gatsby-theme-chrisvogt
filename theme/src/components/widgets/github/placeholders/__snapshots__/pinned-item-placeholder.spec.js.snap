// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Widgets/GitHub/PinnedItemPlaceholder matches the snapshot 1`] = `
<div>
  <Styled(div)>
    <div>
      <RectShape
        color="#efefef"
        style={
          Object {
            "height": 40,
            "marginBottom": "2em",
            "width": 40,
          }
        }
      />
    </div>
    <EmotionCssPropInternal
      __EMOTION_TYPE_PLEASE_DO_NOT_USE__="div"
      css={[Function]}
    >
      <TextBlock
        color="#efefef"
        rows={2}
      />
    </EmotionCssPropInternal>
  </Styled(div)>
  <TextBlock
    color="#efefef"
    rows={3}
  />
</div>
`;
